chainregistry.go

```diff
@@ -475,6 +480,14 @@ func newChainControlFromConfig(cfg *config, chanDB *channeldb.DB,
 			homeChainConfig.Node)
 	}
 
+	// If the no birthday was provided, we will default to using the
+	// beginning of the bitcoin blockchain in case this wallet has yet to be
+	// created. If the wallet has already been created, this value will be
+	// ignored.
+	if walletConfig.Birthday.IsZero() {
```

Comment:
Would appreciate some feedback on the correct behavior here. Other alternatives seem to be using `time.Now()` or just not do anything as it should be ignored.

Reply:
why just not keep it `Zero`?